src/constant.js

```diff
@@ -17,8 +17,10 @@ export const SAMTALEREFERAT_TYPE = 'SAMTALEREFERAT';
 export const STILLING_FRA_NAV_TYPE = 'STILLING_FRA_NAV';
 
 export const INGEN_VALGT = 'INGEN_VALGT';
+export const VENTER = 'VENTER';
```

Comment:
Litt uheldig at vi har både konstanter + type/enum for disse. (teknisk gjeld?)

Reply:
mistenker at dette er pga overgangen mellom js og ts?

Reply:
lar det stå så får vi fikse det med bruk av konstanter en annen gang da det også blir brukt andre steder (litt teknisk gjeld ja).